src/components/application_manager/test/include/application_manager/commands/commands_test.h

```diff
@@ -97,9 +97,10 @@ class CommandsTest : public ::testing::Test {
 
   typedef NiceMock<MockApplicationManagerSettings> MockAppManagerSettings;
   typedef NiceMock<application_manager_test::MockRPCService> MockRPCService;
-  typedef typename TypeIf<kIsNice,
-                          NiceMock<MockApplicationManager>,
-                          MockApplicationManager>::Result MockAppManager;
+  typedef
```

Comment:
@IGapchuk is this an autoformat fix?

Reply:
@AKalinich-Luxoft Yes, it's an autoformat fix. I have added NiceMock wrapper for MockApplicationManager and removed it after that. Added it again in a749f2af0baab2b1647f5f4203a2ee95d9013924.